spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/integration/ReadFromQueryIntegrationTestBase.java

```diff
@@ -118,6 +131,26 @@ public void testQueryOption() {
             .option("query", query)
             .load();
 
+    StructType expectedSchema =
+        DataTypes.createStructType(
+            ImmutableList.of(
+                DataTypes.createStructField("corpus", DataTypes.StringType, true),
+                DataTypes.createStructField("word_count", DataTypes.LongType, true)));
+
+    assertThat(df.schema()).isEqualTo(expectedSchema);
+
+    if (isSpark3AndAbove) {
```

Comment:
It's actually for dsv2 right? Doesn't need to be Spark 3 and above. Can rename to `isDataSourceV2`.

Reply:
it's DSv2 on Spark 3.x ...